### DISCUSSION TITLE
Cant create a JWT signed by private key?

### DISCUSSION THREAD

**Discussion by Scylla2020** [latchset/jwcrypto#259] [General]
Im trying to create a JWT token signed by the private key and cant find a way how.

The docs only have this example for signing but it creates a JWS instead

```
Token = jwt.JWT(header={"alg": "HS256"},
                    claims={"info": "I'm a signed token"})
 Token.make_signed_token(key)
Token.serialize()
```
Is there a way to sign a JWT by private key and return a JWT?

**Comment by simo5** [comment]
That code snippet does give you a JWT:
```
>>> type(Token)
<class 'jwcrypto.jwt.JWT'>
>>> Token.serialize()
'eyJhbGciOiJIUzI1NiJ9.eyJpbmZvIjoiSSdtIGEgc2lnbmVkIHRva2VuIn0.uLTTuSMnZk_Y9vdkbbkTKjVGLsaTcYphEnsqatcnHMI'
```

A signed only JWT is just a JWS.
What are you looking for exactly?
What is missing from the serialized output that you would expect?

**Comment by Scylla2020** [comment]
Oh okay got it, thanks for clarifying . I was just confused by the word JWS, just started learning this today.
